chart/values.yaml

```diff
@@ -0,0 +1,438 @@
+# Default values for airflow.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# User and group of airflow user
+uid: 100
+gid: 101
+
+# Select certain nodes for airflow pods.
+nodeSelector: {}
+affinity: {}
+tolerations: []
+
+# Add common labels to all objects and pods defined in this chart.
+labels: {}
+
+# Ingress configuration
+ingress:
+  # Enable ingress resource
+  enabled: false
+
+  # Enable for cert-manager or kube-lego
+  acme: false
+
+  # Name of tls secret to use on ingress
+  tlsSecretName: ~
+
+  # Annotations always injected when configuring webserver Ingress ojbect
+  webserverAnnotations: {}
+
+  # Annotations always injected when configuring Flower Ingress object
+  flowerAnnotations: {}
+
+  # Base domain for ingress vhosts
+  baseDomain: ~
+
+# Network policy configuration
+networkPolicies:
+  # Enabled network policies
+  enabled: false
+
+# Airflow home directory
+# Used for mount paths
+airflowHome: "/usr/local/airflow"
+
+# Extra annotations to apply to all
+# Airflow pods
+airflowPodAnnotations: {}
+
+# Enable RBAC (default on most clusters these days)
+rbacEnabled: true
+
+# Airflow executor
+# Options: SequentialExecutor, LocalExecutor, CeleryExecutor, KubernetesExecutor
+executor: "KubernetesExecutor"
+
+# If this is true and using LocalExecutor/SequentialExecutor/KubernetesExecutor, the scheudler's
+# service account will have access to communicate with the api-server and launch pods.
+# If this is true and using the CeleryExecutor, the workers will be able to launch pods.
+allowPodLaunching: true
+
+# Default airflow repository
+defaultAirflowRepository: astronomerinc/ap-airflow
+
+# Default airflow tag to deploy
+defaultAirflowTag: 1.10.7-alpine3.10
+
+# Images
+images:
+  airflow:
+    repository: astronomerinc/ap-airflow
+    tag: ~
+    pullPolicy: IfNotPresent
+  flower:
+    repository: astronomerinc/ap-airflow
```

Comment:
I want to use the official airflow image, is it just changing this image? 
Or changes required across the chart?

Reply:
Great point. @dimberman has some experience with this. I believe the intention is to follow up this PR with a better integration into the official image as well as Airflow 2.0. Any thoughts on this Daniel?

Reply:
@schnie I'd say let's merge what we have and I will then 

1. Create a PR making it compatible with the airflow 2.0 official image
2. Create a backport making it compatible with the 1.10 image.

Reply:
I really don't understand how the 'offical' helm chart for airflow could use anything but the offical Dockerfile.
For reference, we already use the offical Dockerfile at [`stable/airflow`](https://github.com/helm/charts/tree/master/stable/airflow)